prac_05/emails.py

```diff
@@ -0,0 +1,14 @@
+email_to_name = {}
+email = input("Email: ")
+while email != "":
+    modified_email = email.replace(",", " ").replace(".", " ")
+    name = " ".join([char for char in (modified_email[:modified_email.find("@")].split()) if not char.isdigit()]).title()
+    confirm = input(f"Is your name {name}? (Y/n)").lower()
+    if confirm == 'y' or "":
```

Comment:
Why use the "no else" pattern here?
`or ""` and `'n' or 'no'` don't do what you (probably) think they do.
https://github.com/CP1401/Practicals/tree/master/prac_03#note

Reply:
I see my error here. Thank you.